docs/src/howto-guides/005_pub_sub.md

```diff
@@ -17,6 +17,13 @@ tedge mqtt pub 'tedge/measurements' '{ "temperature": 21.3 }'
 ```shell
 tedge mqtt pub 'tedge/measurements' '{ "temperature": 21.3 }' --qos 2
 ```
+`tedge mqtt pub` supports publishing a MQTT message with retain flag:
+
+```shell
+tedge mqtt pub --retain 'tedge/measurements' '{ "temperature": 21.3 }'
+```
```

Comment:
It's better to have an example that makes sense

```suggestion
tedge mqtt pub --retain --qos 1 tedge/alarms/critical/high_temperature '{"message": "Temperature is critical"}'
```

Reply:
updated